react-app/src/components/map/MapLayers.tsx

```diff
@@ -1,25 +1,181 @@
+      layers: 'pub:WHSE_ADMIN_BOUNDARIES.ADM_INDIAN_RESERVES_BANDS_SP',
+    },
+    {
+      name: 'First Nation Treaty Areas',
+      url: 'https://openmaps.gov.bc.ca/geo/pub/WHSE_LEGAL_ADMIN_BOUNDARIES.FNT_TREATY_AREA_SP/ows?',
+      layers: 'pub:WHSE_LEGAL_ADMIN_BOUNDARIES.FNT_TREATY_AREA_SP',
+    },
+    {
+      name: 'First Nations Treaty Lands',
+      url: 'https://openmaps.gov.bc.ca/geo/pub/WHSE_LEGAL_ADMIN_BOUNDARIES.FNT_TREATY_LAND_SP/ows?',
+      layers: 'pub:WHSE_LEGAL_ADMIN_BOUNDARIES.FNT_TREATY_LAND_SP',
+    },
+    {
+      name: 'First Nations Treaty Related Lands',
+      url: 'https://openmaps.gov.bc.ca/geo/pub/WHSE_LEGAL_ADMIN_BOUNDARIES.FNT_TREATY_RELATED_LAND_SP/ows?',
+      layers: 'pub:WHSE_LEGAL_ADMIN_BOUNDARIES.FNT_TREATY_RELATED_LAND_S',
+    },
+    {
+      name: 'First Nation Treaty Side Agreements',
+      url: 'https://openmaps.gov.bc.ca/geo/pub/WHSE_LEGAL_ADMIN_BOUNDARIES.FNT_TREATY_SIDE_AGREEMENTS_SP/ows?',
+      layers: 'pub:WHSE_LEGAL_ADMIN_BOUNDARIES.FNT_TREATY_SIDE_AGREEMENTS_SP',
+    },
+  ],
+  landOwnership: [
+    {
+      name: 'Crown Leases',
+      url: 'https://openmaps.gov.bc.ca/geo/pub/WHSE_TANTALIS.TA_CROWN_LEASES_SVW/ows?',
+      layers: 'pub:WHSE_TANTALIS.TA_CROWN_LEASES_SVW',
+    },
+    {
+      name: 'Crown Inventory',
+      url: 'https://openmaps.gov.bc.ca/geo/pub/WHSE_TANTALIS.TA_CROWN_INVENTORY_SVW/ows?',
+      layers: 'pub:WHSE_TANTALIS.TA_CROWN_INVENTORY_SVW',
+    },
+    {
+      name: 'Crown Land Licenses',
+      url: 'https://openmaps.gov.bc.ca/geo/pub/WHSE_TANTALIS.TA_CROWN_LICENSES_SVW/ows?',
+      layers: 'pub:WHSE_TANTALIS.TA_CROWN_LICENSES_SVW',
+    },
+    {
+      name: 'Parcel Boundaries',
+      url: 'https://openmaps.gov.bc.ca/geo/pub/WHSE_CADASTRE.PMBC_PARCEL_FABRIC_POLY_SVW/ows',
+      layers: 'pub:WHSE_CADASTRE.PMBC_PARCEL_FABRIC_POLY_SVW',
+    },
+  ],
+  zoning: [
+    {
+      name: 'Zoning',
+      url: 'https://openmaps.gov.bc.ca/geo/pub/WHSE_LEGAL_ADMIN_BOUNDARIES.OATS_ALR_BOUNDARY_LINES_SVW/ows?',
+      layers: 'pub:WHSE_LEGAL_ADMIN_BOUNDARIES.OATS_ALR_BOUNDARY_LINES_SVW',
+    },
+  ],
+  disturbances: [
+    {
+      name: 'Disturbances',
+      url: 'https://openmaps.gov.bc.ca/geo/pub/WHSE_WASTE.SITE_ENV_RMDTN_SITES_SVW/ows?',
+      layers: 'pub:WHSE_WASTE.SITE_ENV_RMDTN_SITES_SVW',
+    },
+  ],
+};
 
-  return (
-    <>
-      {/* Tile Layer for Street Maps */}
+const MapLayers = () => (
+  <LayersControl position="topleft">
+    <LayersControl.BaseLayer checked name="Street Map">
       <TileLayer url="https://tile.openstreetmap.org/{z}/{x}/{y}.png" />
+    </LayersControl.BaseLayer>
 
-      {/* Parcel Layer for Parcel Boundaries */}
-      <WMSTileLayer
-        url={PARCEL_LAYER_URL}
-        format="image/png; mode=8bit"
-        transparent={true}
-        opacity={0.5}
-        layers="WHSE_CADASTRE.PMBC_PARCEL_FABRIC_POLY_SVW"
-      />
-    </>
-  );
-};
+    <LayersControl.Overlay name="Administrative Boundaries">
```

Comment:
I can see the idea behind setting up these parent components and having the enumerated children nested with a bit of left margin to make them look grouped together but I don't think it's working. It seems like the control displays identically when these parent LayersControl.Overlay's are removed. There are some leaflet plugins out there that provide this kind of functionality if it's really needed.